src/main/resources/data/terrestria/recipes/sakura_leaf_pile.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "minecraft:crafting_shapeless",
```

Comment:
would like to make this a 2x1 recipe (same shape as carpet) that does 2 leaf blocks -> 16 leaf piles, it's the same ratio but it makes the recipe less generic and *closer* to a vanilla recipe